staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -80,6 +80,12 @@ var baseOpts = []VersionedOptions{
 			cel.OptionalTypes(),
 		},
 	},
+	{
+		IntroducedVersion: version.MajorMinor(1, 29),
```

Comment:
I thought here should be 1.28?

Reply:
see https://github.com/kubernetes/kubernetes/pull/118803#discussion_r1238875524

Reply:
I added it back to 1.28 and changed DefaultCompatibilityVersion back to 1.27